test/extended/networking/ipsec.go

```diff
@@ -511,37 +551,17 @@ var _ = g.Describe("[sig-network][Feature:IPsec]", g.Ordered, func() {
 			checkNodeTraffic(v1.IPsecModeDisabled)
 
 			g.By(fmt.Sprintf("configure IPsec in %s mode and validate traffic", mode))
-			// Change IPsec mode to External and packet capture on the node's interface
-			// must be geneve encapsulated ones.
+			// Change IPsec mode to given mode and do packet capture on the node's interface
```

Comment:
This `err := configureIPsecMode(oc, mode)` is not causing a reboot when ipsec machine config extensions are deployed? Is that because we the CI job starts deploys the cluster with IPSec already enabled? In Full mode?

Reply:
yes, this is what we discussed last week. We'll have CI job starting cluster with IPsec full mode and then execute the tests with minimal number of node reboots